modules/health_quest/app/services/health_quest/appointment_service.rb

```diff
@@ -17,11 +17,13 @@ def get_appointments(start_date, end_date, pagination_params = {})
       end
     end
 
-    def get_appointment_by_id(id)
+    def get_appointment_by_id(_id)
```

Comment:
Why are we no longer looking up appointments by id? It would seem this method by its name should do that.

Reply:
I'm open to suggestions. Since we're not using the id argument inside the method body anywhere, rubocop/lint failed. Prefixing it with the underscore was a way to get those checks to pass.

Reply:
I think it's fine if we're goin to use it in the future (e.g. rename it back to `id`).